Dockerfile

```diff
@@ -7,7 +7,8 @@ WORKDIR /build
 COPY . .
 
 ARG GOFLAGS="-ldflags=-w -ldflags=-s"
-RUN CGO_ENABLED=0 go build -o xk6 -trimpath ./cmd/xk6/main.go
+RUN CGO_ENABLED=0 go build -o xk6 -trimpath ./cmd/xk6/main.go && \
+    CGO_ENABLED=0 go build -o xk6-depsync -trimpath ./cmd/xk6-depsync/main.go
```

Comment:
```suggestion
```
This tool is targeting a different kind of User from what the Dockerfile does. I expect the Dockerfile is used mainly by k6 and extensions' users for building new k6 binaries. Instead, this tool will be used by extensions' maintainers where I expect they have Go installed and they could just use the `go install` command.
I would prefer if we start minimal not adding the tool to the Dockerfile and add it only in case there is a real demand from users.

Reply:
Makes sense, it's out now :)